src/tasks/boozedrop.ts

```diff
@@ -389,6 +400,14 @@ export const BoozeDropQuest: Quest = {
         }
         handleCustomPulls("instant_boozeTestPulls", boozeTestMaximizerString);
 
+        if (
+          CommunityService.BoozeDrop.actualCost() > 1 &&
+          StillSuit.distillateModifier("Item Drop") >= 15 &&
+          !get("instant_saveStillsuit", false) &&
+          myInebriety() + 1 < inebrietyLimit()
+        )
+          StillSuit.drinkDistillate();
+
```

Comment:
Is there a logic behind having it as a task above and once again here? If this is intentional, should we also check for the `Buzzed on Distillate` effect before we distillate further (it's quite unlikely that we can if we distillated earlier)